submissions/DmytryjK/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,204 @@
+  },
+  {
+    hero: 'pa',
+    srcHero: 'images/images_cards/pa.jpg'
+  },
+  {
+    hero: 'ember',
+    srcHero: 'images/images_cards/ember.jpg'
+  },
+  {
+    hero: 'lancer',
+    srcHero: 'images/images_cards/lancer.jpg'
+  },
+  {
+    hero: 'gyro',
+    srcHero: 'images/images_cards/gyro.jpg'
+  },
+  {
+    hero: 'necr',
+    srcHero: 'images/images_cards/necr.jpg'
+  },
+  {
+    hero: 'void',
+    srcHero: 'images/images_cards/void.jpg'
+  }
+];
+
+const srcBg = 'images/back/dota_back.svg',
+      winnerPopup = document.querySelector('.winMessage'),
+      mainCardsBlock = document.querySelector('.main__cards-block');
+let doubleElementsOfCards = [];
+
+let parrentCardBlock = document.createElement('ul');
+parrentCardBlock.classList.add('main__cards');
+
+createElementsOfCards();
+
+let currentListCards = document.querySelectorAll('.main__card');
+let activeCards = [],
+  numbersOfCloseCards = 0;
+
+const currentTime = document.querySelector('.timer-current span'),
+      bestTime = document.querySelector('.timer-best span');
+
+let minutes = 0,
+    seconds = 0,
+    bestResultsSheet = [],
+    timerInterval = null;
+
+const buttonToContinue = document.querySelector('.click-to-continue');
+
+setTimeout(showAllCards, 400);
+setTimeout(hideAllCards, 2000);
+
+if (localStorage.getItem('bestTime') !== null) {
+  bestTime.innerText = localStorage.getItem('bestTime');
+  bestResultsSheet.push(bestTime.innerText);
+}
+
+function onClickUserActions(e) {
+  const clickedTarget = e.target.parentNode;
+
+  if (clickedTarget.classList.contains('main__card')) {
+    clickedTarget.classList.add('active-card');
+    activeCards.push(clickedTarget);
+  }
+
+  if (activeCards.length >= 2) {
+
+    if (activeCards[0].getAttribute(["data-name"]) === activeCards[1].getAttribute(["data-name"]) &&
+      activeCards[0].getAttribute(["data-numbers"]) !== activeCards[1].getAttribute(["data-numbers"])) {
+      setTimeout(hideActiveCards, 600, activeCards);
+    } else {
+      setTimeout(closeActiveCards, 800, activeCards);
+    }
+
+    if (numbersOfCloseCards === doubleElementsOfCards.length / 2 - 1) {
+
+      winnerPopup.classList.add('winMessage__active');
+
+      localStorage.setItem('time', currentTime.innerText);
+      currentTime.innerText = localStorage.getItem('time');
+      bestResultsSheet.push(localStorage.getItem('time'));
+
+      if (bestResultsSheet.length > 1) {
+
+        bestTime.innerText = bestResultsSheet.reduce((prevItem, currentItem) => {
+          if (currentItem < prevItem) {
+            return currentItem;
+          } else {
+            return prevItem;
+          }
+        });
+
+      } else {
+        bestTime.innerText = localStorage.getItem('time');
+      }
+
+      localStorage.setItem('bestTime', bestTime.innerText);
+      clearInterval(timerInterval);
+
+      buttonToContinue.addEventListener('click', clickToContinue, false);
+    }
+    activeCards = [];
+  }
+}
+
+function clickToContinue(buttonToContinue) {
```

Comment:
```suggestion
function restartGame(buttonToContinue) {
```
What do you think about that?

Reply:
I think it makes sense to rename the function. The name you suggested better describes what the function does.